genai-perf/genai_perf/profile_data_parser/llm_profile_data_parser.py

```diff
@@ -349,7 +368,11 @@ def _get_input_token_count(self, req_inputs: dict) -> int:
         else:
             raise ValueError(f"Unknown service kind: '{self._service_kind}'.")
 
-        return len(self._tokenizer.encode(input_text))
+        return self._cached_encode(input_text)
+
+    @lru_cache(maxsize=100_000)
```

Comment:
Should we move this to a named constant somewhere instead?

Reply:
Sure, done!